src/components/app/details/triggerView/TriggerView.tsx

```diff
@@ -1161,6 +1195,9 @@ class TriggerView extends Component<TriggerViewProps, TriggerViewState> {
                                 isJobView={this.props.isJobView}
                                 isCITriggerBlocked={nd?.isCITriggerBlocked}
                                 ciBlockState={nd?.ciBlockState}
+                                selectedEnv={this.state.selectedEnv}
```

Comment:
we can remove these state. on reopening the env should be default. Prevent these two states.